csharp/src/ProtocolBuffers/Google.Protobuf.nuspec

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package>
+  <metadata>
+    <id>Google.Protobuf</id>
+    <title>Google Protocol Buffers C#</title>
+    <summary>C# runtime library for Protocol Buffers - Google's data interchange format.</summary>
+    <description>See project site for more info.</description>
+    <version>3.0.0-experimental</version>
```

Comment:
I wasn't really planning to upload the package with this version, this is meant more as a placeholder and the actual version under which we publish is TBD. 
Current version of protobufs is 3.0.0-alpha3, so let's see what will be the version of the next release.

Btw, adding the non-numeric suffix makes the nuget packages marked as pre-release and this has some annoying consequences (like every package that depends on this packages has to be also marked as pre-release).

Should I change the version to something or we leave it like this and determine the right version before we publish?